buid	,	V_9
bus	,	V_26
rtas_pci_ops	,	V_50
out_be32	,	F_25
__iomem	,	T_2
PCIBIOS_BAD_REGISTER_NUMBER	,	V_13
pci_bus	,	V_25
dn	,	V_2
busdn	,	V_28
"ibm,write-pci-config"	,	L_9
dev	,	V_35
len	,	V_46
u32	,	T_1
pci_ext_config_space	,	V_4
first_busno	,	V_47
rtas_pci_read_config	,	F_8
registers	,	V_38
model	,	V_36
state	,	V_16
of_device_is_available	,	F_11
chip_regs	,	V_39
config_access_valid	,	F_1
val	,	V_7
ret	,	V_11
ibm_write_pci_config	,	V_32
resource	,	V_37
CONFIG_EEH	,	F_3
bus_range	,	V_45
"reg0: %x\n"	,	L_5
"Python"	,	L_2
be32_to_cpu	,	F_31
devfn	,	V_20
rtas_call	,	F_5
ioremap	,	F_22
PCIBIOS_SET_FAILED	,	V_18
PCIBIOS_DEVICE_NOT_FOUND	,	V_12
in_be32	,	F_24
phb_set_bus_ranges	,	F_30
eeh_dev_check_failure	,	F_13
"ibm,read-pci-config"	,	L_8
ops	,	V_49
size	,	V_6
returnval	,	V_8
rtas_config_addr	,	F_4
is_python	,	F_17
ibm_read_pci_config	,	V_22
BUID_LO	,	F_7
rtas_write_config	,	F_15
get_phb_buid	,	F_29
pdn_to_eeh_dev	,	F_14
BUID_HI	,	F_6
python_countermeasures	,	F_19
"Can't get address for Python workarounds !\n"	,	L_3
edev	,	V_14
child	,	V_30
rtas_pci_write_config	,	F_16
iounmap	,	F_26
of_address_to_resource	,	F_20
KERN_INFO	,	V_42
last_busno	,	V_48
rtas_token	,	F_28
"read-pci-config"	,	L_6
__be32	,	T_4
printk	,	F_21
pci_controller	,	V_44
rtas_setup_phb	,	F_32
of_get_property	,	F_18
device_node	,	V_27
pci_dn	,	V_1
ulong	,	V_33
found	,	V_29
read_pci_config	,	V_23
rtas_read_config	,	F_2
"write-pci-config"	,	L_7
where	,	V_3
busno	,	V_19
write_pci_config	,	V_34
addr	,	V_10
PRG_CL_RESET_VALID	,	F_23
"bus-range"	,	L_10
"model"	,	L_1
PCI_DN	,	F_10
EEH_PE_CFG_BLOCKED	,	V_17
pci_bus_to_OF_node	,	F_9
EEH_IO_ERROR_VALUE	,	F_12
start	,	V_41
phb	,	V_21
"Python workaround: "	,	L_4
PCIBIOS_SUCCESSFUL	,	V_24
r	,	V_43
sibling	,	V_31
pe	,	V_15
KERN_ERR	,	V_40
__init	,	T_3
init_pci_config_tokens	,	F_27
pdn	,	V_5
